setup.py

```diff
@@ -43,7 +43,7 @@ def has_environment_marker_support():
 
 
 def main():
-    install_requires = ['py>=1.4.33', 'setuptools']  # pluggy is vendored in _pytest.vendored_packages
+    install_requires = ['py>=1.4.33', 'six','setuptools']  # pluggy is vendored in _pytest.vendored_packages
```

Comment:
i do wonder if we should have a minimal version there,
i suspect its necessary once we sort out `_pytest.compat`